src/main/java/org/generationcp/middleware/operation/builder/StandardVariableBuilder.java

```diff
@@ -69,6 +69,17 @@ public StandardVariable create(final int standardVariableId, final String progra
 		return this.getStandardVariableTransformer().transformVariable(variable);
 	}
 
+	public StandardVariable createStandardVariableWithAliasAsName(final int standardVariableId, final String programUUID) {
+
+		final Variable variable = this.getOntologyVariableDataManager().getVariable(programUUID, standardVariableId, false);
+
+		final StandardVariable standardVariable = this.getStandardVariableTransformer().transformVariable(variable);
+		if(variable.getAlias() != null && !variable.getName().isEmpty()) {
```

Comment:
Maybe we can just have a separate field for Alias in StandardVariable, so that we preserve the actual ontology name.